modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/css/OrderableTable.scss

```diff
@@ -27,6 +27,10 @@ html#{$cadmin-selector} .cadmin {
 
 		.orderable-table-row.dragging {
 			opacity: 0.5;
+
+			td.drag-handle-cell {
+				background-color: #528eff !important;
```

Comment:
Is there a clay variable for this? I noticed this in the clay repo, https://github.com/liferay/clay/blob/5b42133dfbde7d1f2141f558202893394a284de8/packages/clay-tabs/docs/markup-tabs.md?plain=1#L1218

Reply:
I looked into this and I'm not sure how to import this clay variable in the file. 
Let me know if you have an idea how to go about doing this.

Reply:
It looks like that value isn't in the clay/css source, only the docs, so I think this should be fine.